cmd/apiserver/main.go

```diff
@@ -59,18 +60,28 @@ func main() {
 	}
 
 	r := mux.NewRouter()
+
 	// Setup rate limiter
-	limitInstance := limiter.New(memory.NewStore(), limiter.Rate{
-		Period: 1 * time.Minute,
-		Limit:  int64(config.RateLimit),
-	}, limiter.WithTrustForwardHeader(true))
-	r.Use(limithttp.NewMiddleware(limitInstance).Handler)
+	store, err := memorystore.New(&memorystore.Config{
+		Tokens:   config.RateLimit,
+		Interval: 1 * time.Minute,
+	})
+	if err != nil {
+		log.Fatalf("failed to create limiter: %v", err)
+	}
+	defer store.Stop()
```

Comment:
I guess more of an upstream comment, does it make sense to have this be a Closer?
https://golang.org/pkg/io/#Closer